backend/src/main/resources/db/migration/V11__add_category_ordinal.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE category ADD COLUMN ordinal BIGINT NOT NULL;
```

Comment:
(member_id, ordinal) 합성 필드에 unique 조건 걸어볼까요??

```suggestion
ALTER TABLE category ADD COLUMN ordinal BIGINT NOT NULL;
ALTER TABLE category ADD CONSTRAINT member_ordinal_unique UNIQUE (member_id, ordinal);
```